charts/taraxa-node/values.yaml

```diff
@@ -94,6 +95,11 @@ node:
     annotationsRpcWS: {}
     annotationsGraphQl: {}
     annotationsGraphQlWS: {}
+    annotationsIndexer:
+      nginx.ingress.kubernetes.io/affinity: "cookie"
+      nginx.ingress.kubernetes.io/session-cookie-name: "stickounet"
+      nginx.ingress.kubernetes.io/session-cookie-expires: "172800"
+      nginx.ingress.kubernetes.io/session-cookie-max-age: "172800"
```

Comment:
I think we should also add `nginx.ingress.kubernetes.io/affinity-mode: persistent` here

Reply:
added